### PR TITLE
Tags specs

### PR DESCRIPTION
**Pull request by Saimis777**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Add / Edit Store
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:2]:
```suggestion
title: Add / Edit tag
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+weight: 1
+---
+
+# Add / Edit tags
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:6]:
```suggestion
# Add / Edit tag
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Add / Edit Store
+weight: 1
+---
+
+# Add / Edit tags 
+
+While adding or editing tags, the following fields/drop downs/parameters are displayed:
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:8]:
```suggestion
While adding or editing a tag, the following fields/drop downs/parameters are displayed:
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+1) Name 
+
+This field is for the name of the tag, the field is mandatory. When trying to save without any name input, shows the message: "The name field is required."
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:12]:
```suggestion
This field allows to fill the name of the tag and is mandatory. If the save button is pressed and if this field is empty, the following error message is displayed: "The name field is required."
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+2) Language
+
+This is a drop-down menu for the language of the tag. The default country (selected in International > Localization > Configuration) is selected by default.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:16]:
```suggestion
This drop-down menu allows selecting the language of the tag. The default country (selected in International > Localization > Configuration) is selected by default.
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:17]: The countries listed in the drop-down are the countries enabled in International > Localization > Languages

File: content/1.7/back-office/shop-parameters/search/tags/index.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/index.md:1]: The "_" is missing before the index.md

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+3) Products
+
+This section has 2 fields, the one on the left indicates all the products that are added to the shop, and has a button below the list named: "Add". When selecting a product, and pressing "add" the selected product moves to the right box, which lists the products that are listed with the selected tag.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:20]:
```suggestion
This section allows to select products associated with the tag and has 2 parts:
- the one on the left lists all the products of the shop
- the one on the right lists all the products selected for the tag.
 When selecting a product, and clicking on "add", the selected product moves to the right box.
The user can select several products and add them to this tag at once.
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+From the field box o the right, you can also select the products, and press "remove" which is below the field, then the product is moved back to the left field box.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:21]:
```suggestion
When selecting a product in the list on the right, and clicking on "remove", the selected product is moved back to the left box.
```

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md:1]: Can you rename content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md by content/1.7/back-office/shop-parameters/search/tags/add-edit-tag.md as you add and edit only one tag, the S should be removed

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 11]
```diff
@@ -0,0 +1,84 @@
+---
+title: Tags listing
+weight: 1
+---
+# Tags listing
+
+The tagging feature enables to associate products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:7]:
```suggestion
The tagging feature allows associating products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for.
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+## Multistore
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:9]: This whole part is wrong, there is no shop association block while creating or editing a tag, so it's added for all shops and then the lists displays all the tags no matter the context

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+## Columns
+
+The page displays the tag list with the following columns:
+
+1. ID
+
+ID number of the tag.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:61]:
```suggestion
ID of the tag.
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+## Columns
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:55]: For this part, the search and sort part are missing

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+Number of products that can be found with the tag.
+
+5. Actions
+
+- **Edit.** This is the main action and it can be performed by clicking on the line or on the button. 
+👉 See the [specs of the add/edit tag](./add-edit-tag.md) 
+
+- **Delete.** The tag can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the  action: "Are you sure you want to delete the selected item(s)?"
+  The action can be canceled by clicking on the cross or on the cancel button.
+  After clicking on "Delete" button, the store is deleted and a successful message is displayed "Successful deletion."
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:82]:
```suggestion
  After clicking on "Delete" button, the tag is deleted and a successful message is displayed "Successful deletion."
```

**Review by marionf — CHANGES_REQUESTED** [review]
I let you see my comments and suggestions, let me know if you have questions

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+title: Tags listing
+weight: 1
+---
+# Tags listing
+
+The tagging feature allows associating products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for through the search bar in front office.
+
+## Multistore
+
+Creating or editing a tag, it's added for all shops and then the lists displays all the tags no matter the context selected.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:11]:
```suggestion
While creating or editing a tag, it's added or edited for all shops (no matter the context selected). So, the list displays all the tags, no matter the context selected.
👉 See [the specs](../../../../../broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-2-add-new-pages-without-shop-association-block)
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+## Columns
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:42]: Could you specify for each columns if a search or a sort is possible ?

File: content/1.7/back-office/shop-parameters/search/tags/add-edit-tag.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+- the one on the right lists all the products selected for the tag.
+When selecting a product in the list on the left, and clicking on "add", the selected product moves to the right box.
+When selecting a product in the list on the right, and clicking on "remove", the selected product is moved back to the left box.
+
+Several products can be selected on the left or on the right box and added or removed at once.
+
+## Call to action
+
+- Save:
+In the case of adding a new tag, the tag is added and a success message is displayed on the tags listing: "Successful creation.". In the case of editing a tag, the tag is edited and a success message is displayed on the tags listing: "Successful update.".
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/add-edit-tag.md:31]:
```suggestion
In the case of adding a new tag and if there is no error, the tag is added and a success message is displayed on the tags listing: "Successful creation.". In the case of editing a tag and if there is no error, the tag is edited and a success message is displayed on the tags listing: "Successful update.".
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+
+- **Edit.** This is the main action and it can be performed by clicking on the line or on the button. 
+👉 See the [specs of the add/edit tag](./add-edit-tag.md) 
+
+- **Delete.** The tag can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the  action: "Are you sure you want to delete the selected item(s)?"
+  The action can be canceled by clicking on the cross or on the cancel button.
+  After clicking on "Delete" button, the tag is deleted and a successful message is displayed "Successful deletion."
+  
+### Bulk actions
+
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:72]:
```suggestion
The Bulk actions button is enabled only if at least one row of the list is selected.
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+weight: 1
+---
+# Tags listing
+
+The tagging feature allows associating products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for through the search bar in front office.
+
+## Multistore
+
+While creating or editing a tag, it's added or edited for all shops (no matter the context selected). So, the list displays all the tags, no matter the context selected.
+👉 See [the specs](../../../../../broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-2-add-new-pages-without-shop-association-block)
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:12]: This link seems wrong, could you fix it ?

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected stores (using the checkboxes). A user can:
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Delete selected: This button will delete the selected stores. 
+When trying to delete the selection without any checkboxes checked, the following message is displayed: "You must select at least one element to delete."
+After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Ok" button, the selected tag is deleted, a successful message is displayed "The selection has been successfully deleted."
+
+## Search
+The search button is disabled as long as there are less than 2 tags in the list. Search can be used for ID, language, name and Products.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:83]:
```suggestion
The search button is disabled as long as there are less than 2 tags in the list. A search can be done by ID, language, name, and Products.
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+It's possible to sort the list by pressing the arrow button near ID, language, name or products.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:84]:
```suggestion
It's possible to sort the list by ID, language, name or products in an ascending or descending way.
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+weight: 1
+---
+# Tags listing
+
+The tagging feature allows associating products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for through the search bar in front office.
+
+## Multistore
+
+While creating or editing a tag, it's added or edited for all shops (no matter the context selected). So, the list displays all the tags, no matter the context selected.
+👉 See [the specs](https://github.com/PrestaShop/prestashop-specs/blob/master/content/1.7/broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-2-add-new-pages-without-shop-association-block)
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:12]: We should use relatives links rather than absolutes links

**Review by marionf — APPROVED** [review]

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+- **Edit.** This is the main action and it can be performed by clicking on the line or on the button. 
+👉 See the [specs of the add/edit tag](./add-edit-tag.md) 
+
+- **Delete.** The tag can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the  action: "Are you sure you want to delete the selected item(s)?"
+  The action can be canceled by clicking on the cross or on the cancel button.
+  After clicking on "Delete" button, the tag is deleted and a successful message is displayed "Successful deletion."
+  
+### Bulk actions
+The Bulk actions button is enabled only if at least one row of the list is selected.
+Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected stores (using the checkboxes). A user can:
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:74]:
```suggestion
Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected tags (using the checkboxes). A user can:
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 25]
```diff
@@ -0,0 +1,87 @@
...
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Delete selected: This button will delete the selected stores.
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:77]:
```suggestion
- Delete selected: This button will delete the selected tags. 
```

File: content/1.7/back-office/shop-parameters/search/tags/tags-listing.md [hunk 26]
```diff
@@ -0,0 +1,87 @@
...
+  The action can be canceled by clicking on the cross or on the cancel button.
+  After clicking on "Delete" button, the tag is deleted and a successful message is displayed "Successful deletion."
+  
+### Bulk actions
+The Bulk actions button is enabled only if at least one row of the list is selected.
+Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected stores (using the checkboxes). A user can:
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Delete selected: This button will delete the selected stores. 
+When trying to delete the selection without any checkboxes checked, the following message is displayed: "You must select at least one element to delete."
```
Comment (marionf) [inline content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:78]:
```suggestion

```

Comment (marionf) [reply content/1.7/back-office/shop-parameters/search/tags/tags-listing.md:78]: On migrated pages, the Bulk actions button is enabled only if at least one row of the list is selected, so this case can't happen

**Review by marwachelly — APPROVED** [review]
PR Can be merged. Thanks @marionf , @Saimis777

**Pull request merged**